include/boost/json/string.hpp

```diff
@@ -380,8 +379,9 @@ class string
         @param sp An optional pointer to the @ref storage
         to use. The container will acquire shared
         ownership of the storage object.
-        If this parameter is omitted, the default storage
-        is used.
+        The default argument for this parameter is `{}`.
+
+        @tparam InputIt The type of the iterators.
```

Comment:
"A type satisfying _InputIterator_" ?

Reply:
This is specified in the constraints

Reply:
Should we bother using `@tparam` then? I think for deduced types, if we are already including it in the constraint what's the point of using tparam?

Reply:
It defines which template parameters exist, which we then constrain.